tests/SolrReindexTest.php

```diff
@@ -259,7 +259,8 @@ public function testRunGroup()
 
         // Test that items in this variant / group are re-indexed
         // 120 divided into 6 groups should be 20 at least (max 21)
-        $this->assertEquals(21, count($ids), 'Group size is about 20', 1);
+        $c = count($ids);
+        $this->assertTrue($c === 20 || $c === 21, 'Group size is about 20', 1);
```

Comment:
I think the `1` parameter was a `$delta` parameter, which doesn't exist anymore. You shouldn't need anymore since you converted the assertion to `assertTrue`.

```suggestion
        $this->assertTrue($c === 20 || $c === 21, 'Group size is about 20');
```

Reply:
Updated